clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -360,14 +360,18 @@ class group_radix_sort {
   }
 
 private:
+  template <typename KT>
+  using key_type =
```

Comment:
How about moving the definition into the function? I don't see it used elsewhere.